core/classes/Misc/ErrorHandler.php

```diff
@@ -126,8 +128,8 @@ public static function catchException(?Throwable $exception, ?string $error_stri
             'LANG_CHARSET' => defined('LANG_CHARSET') ? LANG_CHARSET : 'utf-8',
             'TITLE' => $language->get('errors', 'fatal_error') . ' - ' . $site_name,
             'SITE_NAME' => $site_name,
-            'BOOTSTRAP' => $path . 'vendor/bootstrap/dist/css/bootstrap.min.css',
-            'BOOTSTRAP_JS' => $path . 'vendor/bootstrap/dist/js/bootstrap.min.js',
+            'SEMANTIC_JS' => '/custom/templates/DefaultRevamp/js/fomantic.min.js',
```

Comment:
Might be worth considering adding fomantic elsewhere in case the default revamp template is deleted